spec/lib/serializable_spec.rb

```diff
@@ -60,6 +67,12 @@ class UserSerializer
       users = Array.new(2) { user }
       expect { serialize_and_flatten(users) }.to raise_error(NameError)
     end
+    
+    it "accepts options for conditional attributes" do
+      user = User.new(2, "David", true)
+      user_data = { data: { id: 2, type: :user, name: "David", display_name: "2: David", admin: true } }
```

Comment:
Wouldn't we expect the `admin` attribute to be included in the serialized object?

Reply:
The expected serialized object is `user_data` (line 73). So yes, `admin` is expected. Are you asking why it's not showing up elsewhere?